apps/test/unit/lib/levelbuilder/rubrics/RubricEditorTest.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import LearningGoalItem from '@cdo/apps/lib/levelbuilder/rubrics/LearningGoalItem';
+import Button from '@cdo/apps/templates/Button';
+import {expect} from '../../../../util/reconfiguredChai';
+import RubricEditor from '@cdo/apps/lib/levelbuilder/rubrics/RubricEditor';
+import sinon from 'sinon';
+
+describe('RubricEditorTest ', () => {
+  let wrapper;
+  const addNewConceptSpy = sinon.spy();
+  const deleteItemSpy = sinon.spy();
+  const hndleAiEnabledChangeSpy = sinon.spy();
+  const handleLearningGoalNameChangeSpy = sinon.spy();
+  const sampleLearningGoalList = [
+    {id: 1, learningGoal: 'Goal 1', aiEnabled: true},
+    {id: 2, learningGoal: 'Goal 2', aiEnabled: false},
+    {id: 3, learningGoal: 'Goal 3', aiEnabled: true},
+  ];
+
+  beforeEach(() => {
+    wrapper = shallow(
+      <RubricEditor
+        learningGoalList={sampleLearningGoalList}
+        addNewConcept={addNewConceptSpy}
+        deleteItem={deleteItemSpy}
+        handleAiEnabledChange={hndleAiEnabledChangeSpy}
+        handleLearningGoalNameChange={handleLearningGoalNameChangeSpy}
```

Comment:
nit: you can just stub out the function if you're not going to test it (ie `deletedItem={() => {}}`)

Reply:
If you want to keep them for testing in the future, my other nit is `hndleAiEnabledChangeSpy` should be `handleAiEnabledChangeSpy`